src/test/java/com/example/RaiseTech11/controller/UserControllerTest.java

```diff
@@ -0,0 +1,145 @@
+package com.example.RaiseTech11.controller;
+
+import com.example.RaiseTech11.entity.User;
+import com.example.RaiseTech11.form.CreateForm;
+import com.example.RaiseTech11.form.UpdateForm;
+import com.example.RaiseTech11.service.UserServiceImpl;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.skyscreamer.jsonassert.Customization;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.skyscreamer.jsonassert.comparator.CustomComparator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(UserController.class)
+@ExtendWith(MockitoExtension.class)
+public class UserControllerTest {
+
+    @InjectMocks
+    UserController userController;
+
+    @MockBean
+    UserServiceImpl userServiceImpl;
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Test
+    public void 全てのユーザーのnameを返すこと() throws Exception {
+        List<User> users = new ArrayList<>();
+        users.add(new User(1, "田中太郎", "19901010"));
```

Comment:
このあたりのUserを作成する処理は各テストケースで行われているので、@BeforEachアノテーションなどを使うとDRYに書けます🙌

```suggestion
         @BeforeEach
    public void setUp() {
        users = new ArrayList<>();
        users.add(new User(1, "田中太郎", "19901010"));
        users.add(new User(2, "加藤大", "19951010"));
        users.add(new User(3, "斉藤直樹", "19801010"));
    }
```